### ISSUE TITLE
crypto' is declared but its value is never read

### ISSUE THREAD

**Issue by NRzKICKz** [tradle/react-native-crypto#39]
not sure if im doing something wrong here would love some help .
where im i suppost to put this part ? 
<img width="344" alt="081f2855edf52b80ab6b29ee43fe8bf7" src="https://user-images.githubusercontent.com/37005342/67923139-43e6d680-fb83-11e9-8f3d-b3044ccbc226.png">
import './shim.js'
import crypto from 'crypto'

**Comment by mvayngrib** [comment]
@NRzKICKz did you follow the flow with `rn-nodeify` in the readme?

**Comment by NRzKICKz** [comment]
hello thanks for the help yes i have resolve this problem now im stuck here 
<img width="341" alt="b5bf0ea0031585b97210adeda725de6c" src="https://user-images.githubusercontent.com/37005342/67954028-a01f1a00-fbc6-11e9-9e9b-1b407030636b.png">

**Comment by NRzKICKz** [comment]
hello yes ty yes that has been fix but now im stuck on i have done every step and link. would love some help

bundling failed: Error: Unable to resolve module `react-native-randombytes` from `/Users/macpro/myapp-react-native-dev/node_modules/react-native-crypto/index.js`: Module `react-native-randombytes` does not exist in the Haste module map

**Comment by mvayngrib** [comment]
did u install `react-native-randombytes`?

**Comment by NRzKICKz** [comment]
yes i have also have linked not sure been stuck here for a week would love to see if you have some time to check for me 
<img width="536" alt="c72251090d16fd5487c8c990fc60810b" src="https://user-images.githubusercontent.com/37005342/67969454-bbe2ea00-fbdf-11e9-9e0e-fa65f453b293.png">

**Comment by mvayngrib** [comment]
@NRzKICKz if u want to link a repo, i can take a quick look
